src/session.rs

```diff
@@ -215,6 +221,8 @@ fn aligned_sub_buf(buf: &mut [u8], alignment: usize) -> &mut [u8] {
 
 impl<FS: 'static + Filesystem + Send> Session<FS> {
     /// Run the session loop in a background thread
+    /// # Errors
+    /// Errors if the mount is poisoned.
```

Comment:
Remove this

Reply:
i decided it makes more sense to change the function implementation, than to try to justify the error condition, so I reverted this change.
See PR #405